libcobblersignatures/models/osversion.py

```diff
@@ -68,246 +105,520 @@ def __eq__(self, other):
 
     @property
     def signatures(self) -> list:
+        """
+        This is empty on all known versions. Only here for compatibility reasons currently.
+
+        :setter: Nothing special
+        :getter: Nothing special
+        :deleter: Resets this to an empty list.
+        :type: list
+        """
         return self._signatures
 
     @signatures.setter
     def signatures(self, value: list):
+        """
+        Setter for the signatures.
+
+        :param value: The list with the signatures.
+        :raises TypeError: In case the value was not of of type list.
+        """
         if isinstance(value, list):
             self._signatures = value
         else:
             raise TypeError("Signatures should be a list.")
 
     @signatures.deleter
     def signatures(self):
+        """
+        Resets the value of the signatures to an emtpy list.
+        """
         self._signatures = []
 
     @property
     def version_file(self) -> str:
+        """
+        The regular expression which points to the file with the os-version info.
+
+        :setter: Nothing special
+        :getter: Nothing special
```

Comment:
Same consideration about `TypeError` and better message

Reply:
Fixed in 7f35e0c